### PR TITLE
:bug: - Fix commit order

### PR DESCRIPTION
**Pull request by drinfernoo**
I'm hoping this fixes the `.zip`s not being added before the first commit, so the commits will be more intentional.

### CODE REVIEW

**Comment by inb4after** [comment]
Why not push as a single commit? The history is getting a bit cluttered with `Update metadata files`.

File: .github/workflows/update_modules.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
         run: python ./_repo_generator.py
       - name: Commit packages
         run: |
-          git add *.zip
```
Comment (inb4after) [inline .github/workflows/update_modules.yml:35]: If this doesn't work, the [documentation](https://git-scm.com/docs/git-add#_examples) suggests `git add \*.zip`.

File: .github/workflows/update_modules.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
...
+          git add '*.zip'
```
Comment (inb4after) [inline .github/workflows/update_modules.yml]: If this doesn't work, the [documentation](https://git-scm.com/docs/git-add#_examples) suggests `git add \*.zip`.

Comment (jurialmunkey) [reply .github/workflows/update_modules.yml]: I think it is meant to be `git add ./\*.zip`

**Review by jurialmunkey — CHANGES_REQUESTED** [review]

File: .github/workflows/update_modules.yml (hunk 2 above)
Comment (inb4after) [reply .github/workflows/update_modules.yml]: @jurialmunkey Would that one limit the search to subdirectories only, and ignore the current directory?

Comment (jurialmunkey) [reply .github/workflows/update_modules.yml]: `./` is the current directory
`\*` escapes the wildcard to prevent the shell from expanding it

Comment (jurialmunkey) [reply .github/workflows/update_modules.yml]: Really though I would expect either to work - just if we're following the documentation example to the letter then we should include the current directory explicitly with `./`

**Comment by drinfernoo** [comment]
I decided it makes sense to consolidate these into one commit.

**Comment by jurialmunkey** [comment]
I was going to suggest changing `git add .` to `git add -A` so we also track deletions in case we wanted to add code to the repo generator down the line to clean-up old zips -- but not necessary at the moment.

**Pull request merged**